### PR TITLE
CI: Acceptance tests at PR: Merge with suggested tests

### PR DESCRIPTION
**Pull request by jordimassaguerpla**
## What does this PR change?

Create a "Fast workflow" that runs only the suggested tests to give fast feedback to the user.

## GUI diff

No difference.

- [x] **DONE**

## Documentation
- No documentation needed
- [x] **DONE**

## Test coverage
ℹ️ If a major new functionality is added, it is **strongly recommended** that tests for the new functionality are added to the Cucumber test suite
- No tests
- [x] **DONE**

## Links

Issue(s): https://github.com/SUSE/spacewalk/issues/21015

- [x] **DONE**

## Changelogs

Make sure the changelogs entries you are adding are compliant with https://github.com/uyuni-project/uyuni/wiki/Contributing#changelogs and https://github.com/uyuni-project/uyuni/wiki/Contributing#uyuni-projectuyuni-repository

If you don't need a changelog check, please mark this checkbox:

- [X] No changelog needed

If you uncheck the checkbox after the PR is created, you will need to re-run `changelog_test` (see below)

## Re-run a test

If you need to re-run a test, please mark the related checkbox, it will be unchecked automatically once it has re-run:

- [ ] Re-run test "changelog_test"
- [ ] Re-run test "backend_unittests_pgsql"
- [ ] Re-run test "java_pgsql_tests"
- [ ] Re-run test "schema_migration_test_pgsql"
- [ ] Re-run test "susemanager_unittests"
- [ ] Re-run test "javascript_lint"
- [ ] Re-run test "spacecmd_unittests"

# Before you merge

Check [How to branch and merge properly](https://github.com/uyuni-project/uyuni/wiki/How-to-branch-and-merge-properly)!

### CODE REVIEW

**Comment by srbarrios** [comment]
Code coverage might not be always accurate or updated, so, there is a corner case that I would like that we address differently.

If there are no secondary tests matching on coverage the code changes, then we better run all of them.

Make sense?

**Comment by srbarrios** [comment]
Looks great, it will be good to start trying it.
There are some things we can improve in the future:
- I'd get rid of posting the suggested tests in a comment and then parsing this comment, but instead I'd directly get these lists of tests and filtering them.
- I'd add some user option to skip the filter and run everything if the user wants.

File: testsuite/Rakefile [hunk 1]
```diff
@@ -82,6 +82,40 @@ namespace :utils do
...
+    filter_yaml = args[:filter_yaml]
+    input_yaml = args[:input_yaml]
+    output_yaml = args[:output_yaml]
+    puts "DEBUG"
+    puts "filter_yaml"
+    puts filter_yaml
+    puts "input_yaml"
+    puts input_yaml
+    puts "output yaml"
+    puts output_yaml
```
Comment (srbarrios) [inline testsuite/Rakefile:96]: I guess we want to remove this debugging code at some point

**Comment by srbarrios** [comment]
I think we can close that one. The goal of this PR is already in place.
